R/load_model_metadata.R

```diff
@@ -24,9 +26,9 @@ load_model_metadata <- function(hub_path, model_ids = NULL) {
   UseMethod("load_model_metadata")
 }
 
-
+# cyclomatic complexity of 17 instead of 15 acceptable
```

Comment:
Is this something worth putting in the common `.lintr` file, or is the increased complexity of 17 specific to this file? (_e.g._, `cyclocomp_linter(complexity_limit = 17L)`)

Reply:
I don't think `15L` is particularly meaningful or that `17L` would be more appropriate. I think the particular linter is useful  to bring attention to extra cyclomatic complexity but also ok for the developer to ignore if they feel it's warranted